.travis.yml

```diff
@@ -46,8 +39,15 @@ jobs:
         - wget https://scrutinizer-ci.com/ocular.phar
         - php ocular.phar code-coverage:upload --format=php-clover clover.xml
 
-    - stage: Coding standard
+    - stage: Code Quality
       php: 7.1
+      env: CODING_STANDARD
       install: travis_retry composer install --prefer-dist
       script:
         - ./vendor/bin/phpcs
+
+    - stage: Code Quality
+      php: 7.2
```

Comment:
We should have 7.1 here (the minimum required version). If we leave 7.2, it will force features not available in 7.1.

Reply:
Can you please do a single review I'm done of changing code every second, thank you ^^

Reply:
Tryin' but I always find something new when I go through it again 😅